hazel/third_party/cabal2bazel/bzl/cabal_package.bzl

```diff
@@ -62,13 +63,13 @@ def _get_extra_libs(libnames, extra_libs):
             result.append(lookup)
     return result
 
-def _get_core_dependency_includes(ghc_workspace):
+def _get_core_dependency_includes(name):
```

Comment:
IIRC this is only necessary, because `haskell_import` used to not include headers. Now that it does, I suspect that we can completely drop this part.